dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/action/ViewCMSMaintenanceAction.java

```diff
@@ -244,11 +277,46 @@ else if(cmd.equals("dropoldassets")){
 			boolean dataOnly = Parameter.getBooleanFromString(req.getParameter("dataOnly"), true);
 
 			if(cmd.equals("createZip")) {
+				if(!dataOnly){
+					moveAssetsToBackupDir();
+				}
+				message = "Creating XML Files. ";
+				createXMLFiles();
+				String x = UtilMethods.dateToJDBC(new Date()).replace(':', '-').replace(' ', '_');

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)